packages/component/src/Styles/StyleSet/SendBoxTextarea.js

```diff
@@ -0,0 +1,51 @@
+export default function createSendBoxTextareaStyle({
+  paddingRegular,
+  primaryFont,
+  sendBoxMaxHeight,
+  sendBoxTextColor,
+}) {
+  return {
+    alignItems: 'center',
+    fontFamily: primaryFont,
+    paddingBottom: paddingRegular,
+    paddingLeft: paddingRegular,
+    paddingRight: paddingRegular,
+    paddingTop: paddingRegular,
+
+    '& > div': {
+      fontFamily: 'inherit',
+      fontSize: 'inherit',
+      height: '100%',
+      maxHeight: sendBoxMaxHeight,
+      paddingBottom: 0,
+      paddingTop: 0,
+      position:'relative',
+      width: 'inherit',
+
+      '& > div': {
```

Comment:
doesn't this div `position: absolute`?

Reply:
No, if the inner div is absolute, the containing relative div won’t resize as the inner div grows.